libs/core/async_sycl/tests/unit/common/sycl_vector_add_test_utils.hpp

```diff
@@ -0,0 +1,57 @@
+#include <vector>
```

Comment:
Please add a copyright and license header to this file.

Reply:
Added